src/installer/Directory.Build.props

```diff
@@ -276,7 +284,7 @@
     <CompressedFileExtension Condition="'$(TargetOS)' == 'Windows_NT'">.zip</CompressedFileExtension>
     <CompressedFileExtension Condition="'$(TargetOS)' != 'Windows_NT'">.tar.gz</CompressedFileExtension>
     <InstallerExtension Condition="'$(TargetOS)' == 'Windows_NT'">.msi</InstallerExtension>
-    <InstallerExtension Condition="'$(TargetOS)' == 'OSX'">.pkg</InstallerExtension>
+    <InstallerExtension Condition="'$(TargetOS)' == 'OSX' or '$(TargetOS)' == 'iOS'">.pkg</InstallerExtension>
```

Comment:
(Similar as above) Are we really producing a sharedfx pkg for iOS?

Reply:
No, I was taking a scorched earth approach and can take it away.